Scripts/Web_Scrappers/Github_Scraper/github_scraper.py

```diff
@@ -0,0 +1,38 @@
+import requests
+import json
+
+# Function to call API for details of the User.
+
+
+def findUserInfo(username):
+    url = 'https://api.github.com/users/'+username
+    html = requests.get(url)
+    return json.loads(html.text)
```

Comment:
https://api.github.com/users/<username> return result in JSON so you don't need to convert JSON object explicitly
so json module is not required
```suggestion
    return html.json()
```